fuzzbench/worker.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Self-defined worker module."""
+import time
+
+import redis
+import rq
+
+MAX_TIME_LIMIT = 3600
+REASSIGN_GAP_TIME = 5
+
+
+def no_pending_jobs(queue):
+    """Checks whether the queue has unfinished jobs."""
+    return (queue.deferred_job_registry.count +
+            queue.scheduled_job_registry.count +
```

Comment:
isn't the `scheduled_jobs_registry` contain jobs that were scheduled to be enqueued at a later time by with `enqueue_at()` or `enqueue_in()`? if yes, then we don't need to check that as we only `enqueue()` jobs, right?

Reply:
> the scheduled_jobs_registry contain jobs that were scheduled to be enqueued at a later time by with enqueue_at() or enqueue_in()? 

Yes. I think so. Also refer to https://github.com/google/fuzzbench/pull/588#discussion_r461162784
> if yes, then we don't need to check that as we only enqueue() jobs, right?

Right. Let's change this to `len(queue)`.

Reply:
Done.